context-py/src/lib.rs

```diff
@@ -70,8 +70,32 @@ fn junit_parse(xml: Vec<u8>) -> PyResult<Vec<junit::bindings::BindingsReport>> {
 #[pyfunction]
 fn junit_validate(
     report: junit::bindings::BindingsReport,
-) -> junit::validator::JunitReportValidation {
-    junit::validator::validate(&report.into())
+) -> junit::bindings::BindingsJunitReportValidation {
+    junit::bindings::BindingsJunitReportValidation::from(junit::validator::validate(&report.into()))
+}
+
+#[gen_stub_pyfunction]
+#[pyfunction]
+fn junit_validation_level_to_string(
+    junit_validation_level: junit::validator::JunitValidationLevel,
+) -> String {
+    match junit_validation_level {
+        junit::validator::JunitValidationLevel::Valid => "VALID".to_string(),
+        junit::validator::JunitValidationLevel::SubOptimal => "SUBOPTIMAL".to_string(),
+        junit::validator::JunitValidationLevel::Invalid => "INVALID".to_string(),
+    }
+}
+
+#[gen_stub_pyfunction]
+#[pyfunction]
+fn junit_validation_type_to_string(
+    junit_validation_type: junit::validator::JunitValidationType,
+) -> String {
+    match junit_validation_type {
+        junit::validator::JunitValidationType::Report => "Report".to_string(),
+        junit::validator::JunitValidationType::TestSuite => "TestSuite".to_string(),
+        junit::validator::JunitValidationType::TestCase => "TestCase".to_string(),
+    }
```

Comment:
Can't get stubs for pymethods on enums yet https://github.com/Jij-Inc/pyo3-stub-gen/issues/69, figured these functions would be the path of least resistance

Reply:
I recommend to use a new type pattern to wrap the enum and expose the functions that way. I actually ran into the same issue here:
https://github.com/trunk-io/analytics-cli/pull/176/files#r1857903898

You can see this new type pattern here:
https://github.com/trunk-io/analytics-cli/blob/c21215e6210d2a33cf15c7ed0ec74c5e0f03fcc9/bundle/src/bundle_meta.rs#L113-L147

Reply:
I took a look at this, but I'd need to convert all of the structs that use the current `JunitValidationType` and `JunitValidationLevel` enums to use the new wrapped types. I figured that'd be more work than adding these functions here as a stopgap. Def open to pushback tho!

Reply:
This is fine as-is. I think if we have to support a bunch, probably using newtype pattern will be more scalable